float32[] distance
float32[] forwardSpeed
float32[] angularSpeed
float32[] flipper
